Lombiq.NodeJs.Extensions/package.json

```diff
@@ -24,35 +26,35 @@
     "sass": "^1.52.3",
     "stylelint": "^14.9.1",
     "stylelint-config-standard-scss": "^4.0.0",
-    "terser": "^5.14.1"
+    "terser": "^5.14.1",
+    "textlint": "^12.2.1",
+    "textlint-rule-common-misspellings": "^1.0.1",
+    "textlint-rule-doubled-spaces": "^1.0.0",
+    "textlint-rule-no-todo": "^2.0.1",
+    "textlint-rule-no-zero-width-spaces": "^1.0.1"
   },
   "scripts": {
     "// shell docs": "https://github.com/yarnpkg/berry/blob/master/packages/yarnpkg-shell/tests/shell.test.ts",
     "// advice    ": "When using glob patterns (*/**) as command arguments, wrap the whole argument in quotes.",
     "-": "------------------------------------------------------------------------------------------------------------",
-    "build": "run-p --print-label --continue-on-error \"build:*\"",
-    "compile": "run-p --print-label --continue-on-error \"compile:*\"",
-    "lint": "run-p --print-label --continue-on-error \"lint:*\"",
-    "clean": "run-p --print-label --continue-on-error \"clean:*\"",
-    "watch": "run-p --print-label --continue-on-error \"watch:*\"",
+    "build": "run-p --continue-on-error \"build:*\"",
+    "compile": "run-p --continue-on-error \"compile:*\"",
+    "lint": "run-p --continue-on-error \"lint:*\"",
+    "clean": "run-p --continue-on-error \"clean:*\"",
+    "watch": "run-p --continue-on-error \"watch:*\"",
     "--": "-------------------------------------------------- STYLES -------------------------------------------------",
-    "build:styles":
-      "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _build:styles",
+    "build:styles": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _build:styles",
     "build:styles:args": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _build:styles",
-    "_build:styles": "run-p --print-label --continue-on-error _compile:styles _lint:styles",
+    "_build:styles": "run-p --continue-on-error _compile:styles _lint:styles",
     "lint:styles": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=Assets/Styles pnpm _lint:styles",
     "lint:styles:args": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=$npm_config_source pnpm _lint:styles",
     "_lint:styles": "stylelint \"../../$SCSS_SOURCE/**/*.scss\" || AREA=styles STEP=linting pnpm error",
-    "compile:styles":
-      "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _compile:styles",
+    "compile:styles": "TARGET_DIR=$(node scripts/get-target) SCSS_SOURCE=${SCSS_SOURCE:-Assets/Styles} SCSS_TARGET=${SCSS_TARGET:-wwwroot/css} pnpm _compile:styles",
```

Comment:
While move these to a single, longer line?

Reply:
Its only longer by 15 characters. Even if you break it into two lines the value itself is too long to fit the screen, so there is no advantage to it. Also if you ever try to auto-format the document it will reset it to this style so why fight it?

Reply:
It's so long that we shouldn't really make it even longer. What did auto-format this file for you?

Reply:
Rider did this, but Visual Studio does the same thing too. If you press Ctrl + K, Ctrl + D it becomes a single line.

Reply:
Let's leave this, then.